war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1506,21 +1507,18 @@ function refreshPart(id,url) {
                     Behaviour.applySubtree(node);
                     layoutUpdateCallback.call();
 
-                    if(isRunAsTest) return;
-                    refreshPart(id,url);
+                    if (isRunAsTest && intervalID) {
```

Comment:
The original code had to check the value of `isRunAsTest` both at the end of `refreshPart` and inside `f`, to make sure that the div is updated exactly once if `isRunAsTest` is set; this also had the effect of stopping further updates of the div if `isRunAsTest` was initially `false`, but is later set to `true` after the page is loaded (e.g. from the browser's web console). I didn't want to alter the behavior of the original code at all, hence the extra check. Should I remove it?

If someone needs to stop the updates after the page is loaded, I think it's easier to set a breakpoint inside `f` in the Javascript debugger, see the value of `intervalID` and later call `window.clearInterval` with that value from the web console.